src/System.Security.Cryptography.X509Certificates/tests/CtorTests.cs

```diff
@@ -236,10 +239,16 @@ public static void TestCopyConstructor_Lifetime_Cloned()
             TestPrivateKey(c2, true);
 
             c1.Dispose();
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
```

Comment:
We don't need to keep those calls in with the reference counting approach for proper cleanup of cert files, however the Lifetime_Cloned test only repro'd sporadically (when finalizers were used) unless we added the GC.Collect\Wait. So we could go either way assuming reference counting (with or without GC.Collect\Wait)